### PR TITLE
Ansible templates - dac & usergroup

### PR DESCRIPTION
**Pull request by shawndwells**

### CODE REVIEW

File: shared/templates/audit_rules_usergroup_modification [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# platform = Red Hat Enterprise Linux 7, multi_platform_fedora
+
+# Include source function library.
+. /usr/share/scap-security-guide/remediation_functions
+
+# Perform the remediation for both possible tools: 'auditctl' and 'augenrules'
+
+fix_audit_watch_rule "auditctl" "%PATH%" "wa" "audit_rules_usergroup_modification"
+fix_audit_watch_rule "augenrules" "%PATH%" "wa" "audit_rules_usergroup_modification"
```
Comment (mpreisler) [inline shared/templates/audit_rules_usergroup_modification]: we need to remove the static fixes that shadow this now

**Comment by shawndwells** [comment]
On 8/18/17 2:07 PM, Martin Preisler wrote:
>
> we need to remove the static fixes that shadow this now
>

There aren't any.

RHEL/7/templates/static/bash/audit_rules_usergroup_modification.sh
exists for the old "everything in one rule" check... but nothing was
ever made for the independent/broke out evaluations.

**Comment by shawndwells** [comment]
(there were some DAC audit rules, which are now removed)

**Comment by mpreisler** [comment]
Fails to build: https://jenkins.open-scap.org/job/scap-security-guide-pull-requests/221/label=ssg_new/console

**Comment by shawndwells** [comment]
fixed, i think

`````
$ cmake ../ ; make -j4 rhel7 
-- SCAP Security Guide 0.1.35
-- (see /var/www/html/scap-security-guide/BUILD.md for build instructions)
-- 
-- CMake:
-- generator: Unix Makefiles
-- source directory: /var/www/html/scap-security-guide
-- build directory: /var/www/html/scap-security-guide/build
--  
-- Tools:
-- oscap: /usr/bin/oscap (version: 1.2.14)
-- xsltproc: /usr/bin/xsltproc
-- xmllint: /usr/bin/xmllint
-- xmlwf: /usr/bin/xmlwf
-- sed: /usr/bin/sed
-- shellcheck (optional): SHELLCHECK_EXECUTABLE-NOTFOUND
--  
-- Build options:
-- SSG vendor string: ssgproject
-- OVAL 5.11: ON
-- OVAL schematron validation: ON
-- SVG logo in XCCDFs: ON
--  
-- Products:
-- Chromium: ON
-- Debian 8: ON
-- Fedora: ON
-- Firefox: ON
-- JBoss EAP 5: ON (broken, disabled)
-- JBoss EAP 6: ON
-- JBoss Fuse 6: ON
-- JRE: ON
-- OpenSUSE: ON
-- RHEL-OSP7: ON
-- RHEL 6: ON
-- RHEL 7: ON
-- RHEVM 3: ON (broken, disabled)
-- SUSE 11: ON
-- SUSE 12: ON
-- Ubuntu 14.04: ON
-- Ubuntu 16.04: ON
-- WRLinux: ON
-- Webmin: ON
--  
-- Scanning for dependencies of chromium remediations (bash, ansible, puppet and anaconda)...
-- Scanning for dependencies of chromium checks (OVAL)...
-- Scanning for dependencies of debian8 remediations (bash, ansible, puppet and anaconda)...
-- Scanning for dependencies of debian8 checks (OVAL)...
-- Scanning for dependencies of fedora remediations (bash, ansible, puppet and anaconda)...
-- Scanning for dependencies of fedora checks (OVAL)...
-- Scanning for dependencies of firefox remediations (bash, ansible, puppet and anaconda)...
-- Scanning for dependencies of firefox checks (OVAL)...
-- Scanning for dependencies of eap6 remediations (bash, ansible, puppet and anaconda)...
-- Scanning for dependencies of eap6 checks (OVAL)...
-- Scanning for dependencies of fuse6 remediations (bash, ansible, puppet and anaconda)...
-- Scanning for dependencies of fuse6 checks (OVAL)...
-- Scanning for dependencies of jre remediations (bash, ansible, puppet and anaconda)...
-- Scanning for dependencies of jre checks (OVAL)...
-- Scanning for dependencies of opensuse remediations (bash, ansible, puppet and anaconda)...
-- Scanning for dependencies of opensuse checks (OVAL)...
-- Scanning for dependencies of rhel-osp7 remediations (bash, ansible, puppet and anaconda)...
-- Scanning for dependencies of rhel-osp7 checks (OVAL)...
-- Scanning for dependencies of rhel6 remediations (bash, ansible, puppet and anaconda)...
-- Scanning for dependencies of rhel6 checks (OVAL)...
-- Scanning for dependencies of rhel7 remediations (bash, ansible, puppet and anaconda)...
-- Scanning for dependencies of rhel7 checks (OVAL)...
-- Scanning for dependencies of sle11 remediations (bash, ansible, puppet and anaconda)...
-- Scanning for dependencies of sle11 checks (OVAL)...
-- Scanning for dependencies of sle12 remediations (bash, ansible, puppet and anaconda)...
-- Scanning for dependencies of sle12 checks (OVAL)...
-- Scanning for dependencies of ubuntu1404 remediations (bash, ansible, puppet and anaconda)...
-- Scanning for dependencies of ubuntu1404 checks (OVAL)...
-- Scanning for dependencies of ubuntu1604 remediations (bash, ansible, puppet and anaconda)...
-- Scanning for dependencies of ubuntu1604 checks (OVAL)...
-- Scanning for dependencies of wrlinux remediations (bash, ansible, puppet and anaconda)...
-- Scanning for dependencies of wrlinux checks (OVAL)...
-- Scanning for dependencies of webmin remediations (bash, ansible, puppet and anaconda)...
-- Scanning for dependencies of webmin checks (OVAL)...
-- Configuring done
-- Generating done
-- Build files have been written to: /var/www/html/scap-security-guide/build
[  0%] [  0%] Built target generate-internal-bash-remediation-functions.xml
Built target generate-internal-rhel7-guide.xml
[  0%] Built target generate-internal-rhel7-oval-unlinked.xml
[  0%] Scanning dependencies of target generate-internal-rhel7-bash-remediations.xml
Built target generate-internal-rhel7-anaconda-remediations.xml
[  0%] Built target generate-internal-rhel7-puppet-remediations.xml
[ 16%] Built target generate-internal-rhel7-ansible-remediations.xml
[ 16%] [rhel7-content] generating bash-remediations.xml
[ 16%] Built target generate-internal-rhel7-shorthand.xml
[ 16%] Built target generate-ssg-rhel7-cpe-dictionary.xml
[ 16%] Built target generate-internal-rhel7-xccdf-unlinked-resolved.xml
[ 16%] Built target generate-internal-rhel7-ocil-unlinked.xml
[ 16%] Built target generate-internal-rhel7-xccdf-unlinked-ocilrefs.xml
Unknown target language: "bash"
Merged 700 bash remediations.
[ 16%] Built target generate-internal-rhel7-bash-remediations.xml
Scanning dependencies of target generate-internal-rhel7-xccdf-unlinked.xml
[ 16%] [rhel7-content] generating xccdf-unlinked.xml
[ 16%] Built target generate-internal-rhel7-xccdf-unlinked.xml
Scanning dependencies of target generate-internal-rhel7-linked-xccdf-oval-ocil.xml
[ 16%] [rhel7-content] linking IDs, generating xccdf-linked.xml, oval-linked.xml, ocil-linked.xml
[ 16%] Built target generate-internal-rhel7-linked-xccdf-oval-ocil.xml
Scanning dependencies of target generate-ssg-rhel7-xccdf.xml
Scanning dependencies of target generate-ssg-rhel7-oval.xml
Scanning dependencies of target generate-ssg-rhel7-ocil.xml
[ 16%] [ 16%] [ 16%] [rhel7-content] generating ssg-rhel7-xccdf.xml
[rhel7-content] generating ssg-rhel7-ocil.xml
[rhel7-content] generating ssg-rhel7-oval.xml
[ 16%] Built target generate-ssg-rhel7-ocil.xml
[ 16%] Built target generate-ssg-rhel7-oval.xml
Unselected empty groups in 'standard'.
Unselected empty groups in 'pci-dss'.
Unselected empty groups in 'C2S'.
Unselected empty groups in 'rht-ccp'.
Unselected empty groups in 'common'.
Unselected empty groups in 'stig-rhel7-disa'.
Unselected empty groups in 'stig-rhevh-upstream'.
Unselected empty groups in 'ospp-rhel7'.
Unselected empty groups in 'cjis-rhel7-server'.
Unselected empty groups in 'docker-host'.
Unselected empty groups in 'nist-800-171-cui'.
[ 16%] Built target generate-ssg-rhel7-xccdf.xml
Scanning dependencies of target generate-rhel7-table-stig
Scanning dependencies of target generate-all-roles-rhel7-sh
Scanning dependencies of target generate-ssg-rhel7-xccdf-1.2.xml
Scanning dependencies of target generate-all-roles-rhel7-yml
[ 16%] [ 16%] [ 16%] [ 16%] [rhel7-tables] generating unlinked STIG XCCDF XML file
[rhel7-content] generating ssg-rhel7-xccdf-1.2.xml
[rhel7-roles] generating urn:xccdf:fix:script:sh remediation roles for all profiles in ssg-rhel7-xccdf.xml
[rhel7-roles] generating urn:xccdf:fix:script:ansible remediation roles for all profiles in ssg-rhel7-xccdf.xml
[ 16%] [rhel7-tables] generating HTML STIG test info document
[ 16%] WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
Built target generate-ssg-rhel7-xccdf-1.2.xml
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
Scanning dependencies of target generate-rhel7-table-by-ref-nist
Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-stig-rhevh-upstream.sh' for profile ID 'stig-rhevh-upstream' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:sh.
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-stig-rhel7-disa.sh' for profile ID 'stig-rhel7-disa' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:sh.
[ 16%] Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-pci-dss.sh' for profile ID 'pci-dss' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:sh.
Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-stig-rhel7-disa.yml' for profile ID 'stig-rhel7-disa' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:ansible.
Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-C2S.yml' for profile ID 'C2S' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:ansible.
Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-C2S.sh' for profile ID 'C2S' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:sh.
[rhel7-tables] generating HTML table for nist references
Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-stig-rhevh-upstream.yml' for profile ID 'stig-rhevh-upstream' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:ansible.
Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-pci-dss.yml' for profile ID 'pci-dss' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:ansible.
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
[ 33%] Built target generate-rhel7-table-by-ref-nist
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-standard.sh' for profile ID 'standard' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:sh.
Scanning dependencies of target generate-rhel7-table-by-ref-cis
Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-cjis-rhel7-server.sh' for profile ID 'cjis-rhel7-server' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:sh.
Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-rht-ccp.sh' for profile ID 'rht-ccp' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:sh.
Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-standard.yml' for profile ID 'standard' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:ansible.
[ 33%] Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-cjis-rhel7-server.yml' for profile ID 'cjis-rhel7-server' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:ansible.
Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-rht-ccp.yml' for profile ID 'rht-ccp' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:ansible.
Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-ospp-rhel7.sh' for profile ID 'ospp-rhel7' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:sh.
[rhel7-tables] generating HTML table for cis references
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-ospp-rhel7.yml' for profile ID 'ospp-rhel7' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:ansible.
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-common.sh' for profile ID 'common' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:sh.
[ 33%] Built target generate-rhel7-table-by-ref-cis
WARNING: This content points out to the remote resources. Use `--fetch-remote-resources' option to download them.
WARNING: Skipping https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 file which is referenced from XCCDF content
Scanning dependencies of target generate-rhel7-table-by-ref-cui
Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-docker-host.sh' for profile ID 'docker-host' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:sh.
Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-nist-800-171-cui.sh' for profile ID 'nist-800-171-cui' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:sh.
[ 33%] Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-common.yml' for profile ID 'common' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:ansible.
Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-docker-host.yml' for profile ID 'docker-host' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:ansible.
[rhel7-tables] generating HTML table for cui references
Generated '/var/www/html/scap-security-guide/build/roles/ssg-rhel7-role-nist-800-171-cui.yml' for profile ID 'nist-800-171-cui' in benchmark 'RHEL-7', template=urn:xccdf:fix:script:ansible.
[ 33%] Built target generate-all-roles-rhel7-sh
[ 33%] Built target generate-all-roles-rhel7-yml
Scanning dependencies of target generate-rhel7-table-by-ref-pcidss
Scanning dependencies of target generate-rhel7-table-nistrefs-common
[ 33%] [ 33%] Built target generate-rhel7-table-by-ref-cui
[rhel7-tables] generating HTML table for pcidss references
[ 33%] Scanning dependencies of target generate-rhel7-table-nistrefs-ospp-rhel7
[rhel7-tables] generating HTML NIST refs table for common profile
[ 33%] [ 33%] Built target generate-rhel7-table-by-ref-pcidss
[rhel7-tables] generating HTML NIST refs table for ospp-rhel7 profile
Scanning dependencies of target generate-rhel7-table-nistrefs-C2S
[ 33%] [rhel7-tables] generating HTML NIST refs table for C2S profile
[ 33%] [rhel7-tables] generating HTML STIG table
[ 33%] Built target generate-rhel7-table-stig
[ 50%] Built target generate-rhel7-table-nistrefs-common
Scanning dependencies of target generate-rhel7-table-nistrefs-stig-rhel7-disa
Scanning dependencies of target generate-rhel7-table-cces
[ 50%] [ 50%] [rhel7-tables] generating HTML NIST refs table for stig-rhel7-disa profile
[rhel7-tables] generating HTML CCE identifiers table
[ 50%] Built target generate-rhel7-table-nistrefs-C2S
[ 50%] Built target generate-rhel7-table-cces
Scanning dependencies of target generate-rhel7-table-srg
[ 50%] Built target rhel7-roles
[ 50%] [rhel7-tables] generating HTML SRG map table (flat=no)
Scanning dependencies of target generate-ssg-rhel7-pcidss-xccdf-1.2.xml
[ 66%] [rhel7-content] building ssg-rhel7-pcidss-xccdf-1.2.xml from ssg-rhel7-xccdf-1.2.xml (PCI-DSS centered benchmark)
[ 66%] Built target generate-rhel7-table-nistrefs-ospp-rhel7
[ 66%] [rhel7-tables] generating HTML SRG map table (flat=yes)
ERROR:Rule 'xccdf_org.ssgproject.content_rule_service_auditd_enabled' references PCI-DSS 'Req-10' but doesn't match any Group ID in our requirement tree. Perhaps it's referencing something we don't consider applicable on the Operating System level?
WARNING:716 rules don't reference PCI-DSS!
[ 66%] Built target generate-rhel7-table-nistrefs-stig-rhel7-disa
[ 66%] Built target generate-ssg-rhel7-pcidss-xccdf-1.2.xml
Scanning dependencies of target generate-ssg-rhel7-ds.xml
[ 66%] [rhel7-content] generating ssg-rhel7-ds.xml
No extended-components, nothing to do...
[ 83%] Built target generate-ssg-rhel7-ds.xml
[ 83%] Built target rhel7-content
Scanning dependencies of target generate-ssg-rhel7-guide-index.html
[ 83%] [rhel7-guides] generating HTML guides for all profiles in ssg-rhel7-ds.xml
Generated '/var/www/html/scap-security-guide/build/guides/ssg-rhel7-guide-C2S.html' for profile ID 'xccdf_org.ssgproject.content_profile_C2S' in benchmark 'xccdf_org.ssgproject.content_benchmark_RHEL-7'.
[100%] Built target generate-rhel7-table-srg
Generated '/var/www/html/scap-security-guide/build/guides/ssg-rhel7-guide-cjis-rhel7-server.html' for profile ID 'xccdf_org.ssgproject.content_profile_cjis-rhel7-server' in benchmark 'xccdf_org.ssgproject.content_benchmark_RHEL-7'.
Generated '/var/www/html/scap-security-guide/build/guides/ssg-rhel7-guide-common.html' for profile ID 'xccdf_org.ssgproject.content_profile_common' in benchmark 'xccdf_org.ssgproject.content_benchmark_RHEL-7'.
[100%] Built target rhel7-tables
Generated '/var/www/html/scap-security-guide/build/guides/ssg-rhel7-guide-stig-rhel7-disa.html' for profile ID 'xccdf_org.ssgproject.content_profile_stig-rhel7-disa' in benchmark 'xccdf_org.ssgproject.content_benchmark_RHEL-7'.
Generated '/var/www/html/scap-security-guide/build/guides/ssg-rhel7-guide-rht-ccp.html' for profile ID 'xccdf_org.ssgproject.content_profile_rht-ccp' in benchmark 'xccdf_org.ssgproject.content_benchmark_RHEL-7'.
Generated '/var/www/html/scap-security-guide/build/guides/ssg-rhel7-guide-pci-dss.html' for profile ID 'xccdf_org.ssgproject.content_profile_pci-dss' in benchmark 'xccdf_org.ssgproject.content_benchmark_RHEL-7'.
Generated '/var/www/html/scap-security-guide/build/guides/ssg-rhel7-guide-stig-rhevh-upstream.html' for profile ID 'xccdf_org.ssgproject.content_profile_stig-rhevh-upstream' in benchmark 'xccdf_org.ssgproject.content_benchmark_RHEL-7'.
Generated '/var/www/html/scap-security-guide/build/guides/ssg-rhel7-guide-docker-host.html' for profile ID 'xccdf_org.ssgproject.content_profile_docker-host' in benchmark 'xccdf_org.ssgproject.content_benchmark_RHEL-7'.
Generated '/var/www/html/scap-security-guide/build/guides/ssg-rhel7-guide-standard.html' for profile ID 'xccdf_org.ssgproject.content_profile_standard' in benchmark 'xccdf_org.ssgproject.content_benchmark_RHEL-7'.
Generated '/var/www/html/scap-security-guide/build/guides/ssg-rhel7-guide-default.html' for profile ID '' in benchmark 'xccdf_org.ssgproject.content_benchmark_RHEL-7'.
Generated '/var/www/html/scap-security-guide/build/guides/ssg-rhel7-guide-nist-800-171-cui.html' for profile ID 'xccdf_org.ssgproject.content_profile_nist-800-171-cui' in benchmark 'xccdf_org.ssgproject.content_benchmark_RHEL-7'.
Generated '/var/www/html/scap-security-guide/build/guides/ssg-rhel7-guide-ospp-rhel7.html' for profile ID 'xccdf_org.ssgproject.content_profile_ospp-rhel7' in benchmark 'xccdf_org.ssgproject.content_benchmark_RHEL-7'.
Generated '/var/www/html/scap-security-guide/build/guides/ssg-rhel7-PCIDSS-RHEL-7-guide-pci-dss_centric.html' for profile ID 'xccdf_org.ssgproject.content_profile_pci-dss_centric' in benchmark 'xccdf_org.ssgproject.content_benchmark_PCIDSS-RHEL-7'.
Generated '/var/www/html/scap-security-guide/build/guides/ssg-rhel7-PCIDSS-RHEL-7-guide-default.html' for profile ID '' in benchmark 'xccdf_org.ssgproject.content_benchmark_PCIDSS-RHEL-7'.
[100%] Built target generate-ssg-rhel7-guide-index.html
[100%] Built target rhel7-guides
[100%] Built target rhel7

`````

**Comment by shawndwells** [comment]
@openscap-ci test this please

**Comment by shawndwells** [comment]
new build: https://jenkins.open-scap.org/job/scap-security-guide-pull-requests/224/

**Comment by shawndwells** [comment]
Since it's Friday and everyone left for the weekend, switching this back to WIP so I can tinker with the rest of the auditing templates

**Comment by dahaic** [comment]
Jenkins test this please.

**Comment by mpreisler** [comment]
NACK. Rebase, clean up the commits. There is no need to have any merge commits in a PR.

**Comment by shawndwells** [comment]
Don't have the time. Someone else will need to create these playbooks.

**Comment by mpreisler** [comment]
Recovered into https://github.com/mpreisler/scap-security-guide/tree/shawn_2267

Will open a new PR when I get some extra time.

**Comment by mpreisler** [comment]
I would rebase and fix conflicts but this is fairly problematic. The same template was implemented by @WesleyCeraso so there is a lot of conflicts. I may crawl through this one by one and figure out what's missing but don't have time for that today.

**Pull request closed without merging**